src/bank1.asm

```diff
@@ -6773,83 +6780,97 @@ label_6E03::
 label_6E18::
     ret
 
-label_6E19::
-    db $C6, $C2, $C0, $C2, $F0, $B5, $A7, $28, 6, $3D, $E0, $B5, $C3, $B1, $6E
+IntroSeaPaletteTable::
+    db $C6, $C2, $C0, $C2
 
-label_6E28::
-    ld   a, [$FFCC]
-    and  $80
-    jp   z, label_6EB1
+IntroHandlerEntryPoint::
```

Comment:
NB: this is the entry point for the introduction sequence, called from the main dispatch table.